test/e2e/tests/releases/utils/methods.ts

```diff
@@ -0,0 +1,211 @@
+import {type SanityClient, type SanityDocument} from '@sanity/client'
+
+export const CLIENT_OPTIONS = {
+  apiVersion: 'v2025-02-19',
+}
+
+export const createRelease = async ({
+  sanityClient,
+  dataset,
+  releaseId,
+  metadata,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+  metadata: Record<string, any>
+}) => {
+  await sanityClient.withConfig(CLIENT_OPTIONS).request({
+    uri: `/data/actions/${dataset}`,
+    method: 'POST',
+    body: {
+      actions: [
+        {
+          actionType: 'sanity.action.release.create',
+          releaseId: releaseId,
+          metadata: metadata,
+        },
+      ],
+    },
+  })
+}
+
+export const archiveRelease = async ({
+  sanityClient,
+  dataset,
+  releaseId,
+}: {
+  sanityClient: SanityClient
+  dataset: string | undefined
+  releaseId: string
+}) => {
+  const query = `*[_type == "system.release" && _id == "_.releases.${releaseId}"][0]`
+  const release = await sanityClient.fetch(query)
+
+  if (release.state === 'archived') {
```

Comment:
Is this an error situation that we should throw in? Doesn't seem like we should be calling `archiveRelease` on an already archived release.

Reply:
I think this was part of a previous attempt at debug. Let me come back to you and if I don't have a good answer I'll remove it 👍

Reply:
Removed, thank you for catching this 🙏